api/spec/requests/spree/api/return_authorizations_controller_spec.rb

```diff
@@ -17,8 +17,14 @@ module Spree
       it "can create a new return authorization" do
         stock_location = FactoryBot.create(:stock_location)
         reason = FactoryBot.create(:return_reason)
+        reimbursement = FactoryBot.create(:reimbursement_type)
+        unit = FactoryBot.create(:inventory_unit)
         rma_params = { stock_location_id: stock_location.id,
                        return_reason_id: reason.id,
+                       return_items_attributes: [{
+                         inventory_unit_id: unit.id,
+                         preferred_reimbursement_type_id: reimbursement.id,
+                       }],
```

Comment:
@ibudiallo I can see you changed the setup part of the example, adding also the `preferred_reimbursement_type_id` parameter, but I think the feature is not actually exercised in any of the subsequent expectations. 

I think adding something like: 
```ruby
        return_authorization = Spree::ReturnAuthorization.last
        expect(return_authorization.return_items.first.preferred_reimbursement_type).to eql reimbursement
```
would ensure that the new code is actually working, what do you think?

Reply:
Completely make sense. I've added the test now.